app/package/controllers.py

```diff
@@ -330,70 +331,66 @@ def purge_data_package(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
-@package_blueprint.route("/<publisher>/<package>/finalize",
-                         methods=["POST"])
+@package_blueprint.route("/upload", methods=["POST"])
 @requires_auth
-@is_allowed('Package::Create')
-def finalize_metadata(publisher, package):
+def finalize_metadata():
     """
     DPR metadata finalize operation.
     This API is responsible for getting data from S3 and push it to RDS.
```

Comment:
Not just getting data but getting the Data Package (datapackage.json, README) and importing it into our MetaStore

(and later on we may import the data into a DataStore!)